services/hooks/schemas/constants.yml

```diff
@@ -1,6 +1,7 @@
 identifier-pattern:     "^([a-zA-Z0-9-_]*)$"
 identifier-min-length:  1
 identifier-max-length:  22
+identifier-extended-max-length:  40
```

Comment:
If we're switching to a longer format, do we need to keep the old length around?  All old IDs are valid new IDs and my understanding is that this patch is designed to update *everything* to the new format in one go.

Could we have a single field for max identifier length?  If not, maybe we should hold off on landing until we can have a single field?  I'm concerned that the old identifier (indentifier-max-length) will end up living in perpetuity and will be confusing to future implementers.